concepticondata/conceptlists/Kornai-2018-1400.tsv

```diff
@@ -1060,9 +1060,9 @@ Kornai-2018-1400-1058	1058	2951	series	N			sorozat	series	seria	structure, HAS i
 Kornai-2018-1400-1059	1059	1422	serious	A			komoly	gravis	powaźny	deep(feeling), bad, lack(joke), lack(pretend)
 Kornai-2018-1400-1060	1060	2243	serve	V	3648	SERVE (SOMEONE)	szolgál	servio	słuźyć	work, FOR =PAT, <CAUSE[food AT/2744 =PAT]>
 Kornai-2018-1400-1061	1061	2244	service	N			szolgálat	officium	słuźba	=AGT[person, work], work FOR/2782 person[other]
-Kornai-2018-1400-1062	1062	1364	set	V			kitűz	constituo	ustalać	
+Kornai-2018-1400-1062	1062	1364	set	V	160	SET (HEAVENLY BODIES)	kitűz	constituo	ustalać	
 Kornai-2018-1400-1063	1063	2746	set	N			kollekció	classis	kolekcja	group, HAS many(item), together, unit, item HAS common(characteristic)
-Kornai-2018-1400-1064	1064	2375	set	V			tesz	pono	kłaść	=AGT CAUSE[=PAT AT/2744 position[<stable>,<proper>]]
+Kornai-2018-1400-1064	1064	2375	set	V	160	SET (HEAVENLY BODIES)	tesz	pono	kłaść	=AGT CAUSE[=PAT AT/2744 position[<stable>,<proper>]]
```

Comment:
See comment above.

Reply:
@martino-vic Could you help with this gloss? Do you think it fits?

Reply:
Given a Google search of the glosses, this needs to be unmapped.